setup.py

```diff
@@ -30,7 +30,6 @@
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 3',
```

Comment:
You could add `'Programming Language :: Python :: 3 :: Only'` here.

Reply:
Done.